.github/CODEOWNERS

```diff
@@ -4,63 +4,66 @@
 # The last matching pattern takes precedence.
 # https://help.github.com/articles/about-codeowners/
 
-*                                                   @backstage/community-plugins-maintainers
-yarn.lock                                           @backstage/community-plugins-maintainers @backstage-service
-*/yarn.lock                                         @backstage/community-plugins-maintainers @backstage-service
+                    *                                                   @backstage/community-plugins-maintainers
```

Comment:
Is this change intentional?

Reply:
Sorry, which change? I did format the entire file to be more readable so the right hand column is properly lined up

Reply:
Ah, I see it now, should be fixed 👍